docs/clients/dot-net.md

```diff
@@ -0,0 +1,17 @@
+# Welcome to MkDocs
+
+For full documentation visit [mkdocs.org](http://mkdocs.org).
+
+## Commands
+
+* `mkdocs new [dir-name]` - Create a new project.
+* `mkdocs serve` - Start the live-reloading docs server.
+* `mkdocs build` - Build the documentation site.
+* `mkdocs help` - Print this help message.
+
+## Project layout
+
+    mkdocs.yml    # The configuration file.
+    docs/
+        index.md  # The documentation homepage.
+        ...       # Other markdown pages, images and other files.
```

Comment:
Yes, this is just a placeholder for now, just so the item shows up in the Table of Contents. I will make sure to change it so that it just says something like [THIS IS A PLACEHOLDER FOR FUTURE CONTENT]